[2025-05-25T09:21:37.834+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-25T09:21:37.853+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_etl_pipeline.load_weather_data manual__2025-05-25T09:21:24.165774+00:00 [queued]>
[2025-05-25T09:21:37.865+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_etl_pipeline.load_weather_data manual__2025-05-25T09:21:24.165774+00:00 [queued]>
[2025-05-25T09:21:37.865+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-05-25T09:21:37.891+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_weather_data> on 2025-05-25 09:21:24.165774+00:00
[2025-05-25T09:21:37.894+0000] {standard_task_runner.py:63} INFO - Started process 1351 to run task
[2025-05-25T09:21:37.897+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'weather_etl_pipeline', 'load_weather_data', 'manual__2025-05-25T09:21:24.165774+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/etl_weather.py', '--cfg-path', '/tmp/tmp0ealm0b5']
[2025-05-25T09:21:37.899+0000] {standard_task_runner.py:91} INFO - Job 43: Subtask load_weather_data
[2025-05-25T09:21:37.955+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_etl_pipeline.load_weather_data manual__2025-05-25T09:21:24.165774+00:00 [running]> on host fa09eeb0c661
[2025-05-25T09:21:38.052+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='weather_etl_pipeline' AIRFLOW_CTX_TASK_ID='load_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-25T09:21:24.165774+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-25T09:21:24.165774+00:00'
[2025-05-25T09:21:38.053+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-25T09:21:38.082+0000] {base.py:84} INFO - Using connection ID 'mysql_weather_api' for task execution.
[2025-05-25T09:21:38.143+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-05-25T09:21:38.144+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-25T09:21:38.153+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_etl_pipeline, task_id=load_weather_data, run_id=manual__2025-05-25T09:21:24.165774+00:00, execution_date=20250525T092124, start_date=20250525T092137, end_date=20250525T092138
[2025-05-25T09:21:38.190+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-25T09:21:38.207+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-25T09:21:38.209+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
